draft-thomson-webpush-protocol.xml

```diff
@@ -315,16 +303,20 @@ Host: push.example.net
 ]]></artwork>
       </figure>
       <t>
-        A response with a 201 (Created) status code includes a URI for the
-        "subscription" resource in the Location header field.
+        A response with a 201 (Created) status code MUST include a URI for the
+        new "subscription" resource in the Location header field.
+      </t>
+      <t>
+        The push server MUST also return new "push" and "receipt subscribe"
```

Comment:
I've added the more specific language for the link relations.
I thought the intent for forcing the AS to subscribe for receipts was to allow it to create multiple unique channels for listening? Otherwise, the original proposal to simply return a pre-subscribed resource is simpler?